bin/tronctl

```diff
@@ -129,6 +133,24 @@ def control_objects(args):
         yield edit(args, tron_id.url)
 
 
+def move(args):
+    try:
+        old_name = args.id[0]
+        new_name = args.id[1]
+    except IndexError as e:
+        raise SystemExit(f"Error: Move command needs two arguments.\n{e}")
+
+    tron_client = client.Client(args.server)
+    url_index = tron_client.index()
+    job_index = url_index['jobs']
+    if old_name not in job_index.keys():
```

Comment:
These checks should happen in the controller. If someone hits the API directly, they could bypass these checks and cause an error.

Reply:
OK, I will keep these checks here to report the error as early as possible. And I'll add the check at controller too.